docs/variant-specific/pink.mdx

```diff
@@ -253,24 +256,67 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
 
 <BubblegumFinesse />
 
-- _Bubblegum Finesses_ can only be used to get pink cards.
+### The Bubblegum Prompt
+
+- Normally, you can only _Prompt_ a card if it is the left-most clued card. However, you can _Prompt_ a non-left-most clued card by violating _Pink Promise_. This is called a _Bubblegum Prompt_, which is conceptually similar to a _Bubblegum Finesse_.
+- For example, in a 4-player game:
+  - All the 1's are played on the stacks.
+  - Bob's _Finesse Position_ card has negative pink. He also has three cards clued with number 5 on slot 2, slot 3, and slot 4.
+  - Alice clues number 3 to Donald, which focuses a pink 4 on slot 2 and touches a pink 3 on slot 3.
+  - Bob sees that in order to make the pink 3 and the pink 4 playable, someone has to have the pink 2. Since no-one else seems to have it, Bob must have it as one of his clued 5's.
+  - Given that Alice violated _Pink Promise_, Bob can determine that his pink 2 is on his slot 3. Bob plays his slot 3, and it is the pink 2.
+  - Cathy discards.
+  - Donald knows that normally, clues do not cause non-left-most cards to become _Prompted_. Thus, Donald knows that Bob must be responding to Alice's _Bubblegum Prompt_ on a card that is not actually a 3.
+  - The next card after a pink 3 is a pink 4, so Donald knows that the card that Alice focused is a pink 4. Subsequently, since Alice's clue was a _Play Clue_, the pink 4 must be playable. In order for a pink 4 to be playable, someone has to have a pink 3. Donald does not see it anywhere else, so he determines that he must have the pink 3 on slot 3 (since that is the only other clued card that could possibly be a pink 3).
```

Comment:
I'm trying to workshop ways to simplify/condense the statements. The first sentence here sounds a bit weird as it's an obvious statement, but it's not clear how it immediately relates to the bubblegum logic.

"Thus, Donald knows that the focus of Alice's _Play Clue_ cannot be a 3; it must be a pink 4. In order for the 4 to be playable, someone must have a pink 3. ..."